src/data/filterConstants.js

```diff
@@ -0,0 +1,302 @@
+export const officeRegions = [
+  {
+    code: 'zar',
+    location: { city: 'Arlington', state: 'VA' },
+    regions: [
+      'Alabama',
+      'Georgia',
+      'Maryland',
+      'North Carolina',
+      'Pennsylvania (26 western counties)',
+      'South Carolina',
+      'Virginia',
+      'West Virginia',
+    ],
+  },
+  {
+    code: 'zbo',
+    location: { city: 'Boston', state: 'MA' },
+    regions: ['Maine', 'Massachusetts', 'New Hampshire', 'Rhode Island'],
+  },
+  {
+    code: 'zch',
+    location: { city: 'Chicago', state: 'IL' },
+    regions: [
+      'Illinois',
+      'Indiana',
+      'Ohio',
+      'Michigan',
+      'Wisconsin',
+      'Minnesota',
+      'North Dakota',
+      'South Dakota',
+      'Montana',
+      'Idaho',
+      'Nebraska',
+      'Kansas',
+      'Iowa',
+      'Missouri',
+      'Kentucky',
+    ],
+  },
+  {
+    code: 'zhn',
+    location: { city: 'Houston', state: 'TX' },
+    regions: ['Colorado', 'Oklahoma', 'New Mexico', 'Texas', 'Utah', 'Wyoming'],
+  },
+  {
+    code: 'zla',
+    location: { city: 'Los Angeles', state: 'CA' },
+    regions: [
+      'Arizona',
+      'California (9 southern counties)',
+      'Guam',
+      'Hawaii',
+      'Nevada (4 southern counties)',
+    ],
+  },
+  {
+    code: 'zmi',
+    location: { city: 'Miami', state: 'FL' },
+    regions: ['Florida', 'Puerto Rico', 'US Virgin Islands'],
+  },
+  {
+    code: 'znk',
+    location: { city: 'Newark', state: 'NJ' },
+    regions: [
+      'Connecticut',
+      'Delaware',
+      'New Jersey',
+      'New York (Manhattan + 50 northern counties)',
+      'Pennsylvania(41 eastern counties)',
+      'Vermont',
+    ],
+  },
+  {
+    code: 'zny',
+    location: { city: 'Bethpage', state: 'NY' },
+    regions: ['New York (12 southern counties, excluding Manhattan)'],
+  },
+  {
+    code: 'zol',
+    location: { city: 'New Orleans', state: 'LA' },
+    regions: ['Louisiana', 'Arkansas', 'Mississippi', 'Tennessee'],
+  },
+  {
+    code: 'zsf',
+    location: { city: 'San Francisco', state: 'CA' },
+    regions: [
+      'Alaska',
+      'California (49 northern counties)',
+      'Nevada (13 northern counties)',
+      'Oregon',
+      'Washington',
+    ],
+  },
+];
+
+export const continentEnum = Object.freeze({
+  africa: 'Africa',
+  asia: 'Asia',
+  europe: 'Europe',
+  northAmerica: 'North America',
+  oceania: 'Oceania',
+  southAmerica: 'South America',
+  unknown: 'Unknown',
+});
+
```

Comment:
Great use of an enum.  Definitely agree that it was a great implementation to avoid typos